src/elements/components/LoginForm.tsx

```diff
@@ -30,76 +31,93 @@ export const authState = {
   onLogin: () => {},
   onLogout: () => {}
 };
+export let getAuthState: () => typeof defaultValues = () => defaultValues;
+export let setAuthState: (updatedProps: any) => void = () => {};
 
-const FeatheryAuthGate = ({
+const LoginForm = ({
```

Comment:
LoginProvider? i know we've done a bunch of renames but maybe that will feel more "gatey"